src/CMakeLists.txt

```diff
@@ -198,6 +198,7 @@ set(
     lib/simulation/simulated_map.cpp
     lib/simulation/simulated_positioner.cpp
     lib/simulation/spawner.cpp
+    lib/simulation/gps_trace.cpp
```

Comment:
Questi file vengono tenuti in ordine alfabetico, ma prima di riordinare qui, io sposterei i file di gps_trace dalla cartella simulation/ (che contiene componenti FCPP per la simulazione) alla cartella data/ (che contiene tipi di dato che vengono usati dai programmi aggregati)